### ISSUE TITLE
Arastta Database version info/check/fix in update page

### ISSUE THREAD

**Issue by syntaxerror** [arastta/arastta#636]
The database check, and possibility to add missing updates, could go into the update page.

«Database version
Your database isn't up to date - Click here to fix it»

Something like Joomla already have: https://docs.joomla.org/Help36:Extensions_Extension_Manager_Database

Because, shit just happens sometimes: https://arastta.org/forum/slide-show-issues#reply-2410

:)

**Comment by denisdulici** [comment]
Good idea, must consider this for 1.6 version.

**Comment by arnisjuraga** [comment]
You can use, at least, some SQL "migration" tool. Otherwise - after pulling latest code I get:

```Notice: Error: Table 'arastta_indeed_p.pbr_user_activity' doesn't exist```

So ho do I fix it? I mean, I can, of course, create the table manually. But I am not sure, this is the only change in latest code. So, do you have any .sql version history where to take ".sql" changes from?

We are using Phinx.

**Comment by syntaxerror** [comment]
Well, that is quite a different matter as you are supposed to do updates, not code pulls: https://arastta.org/docs/update

If you pull the code, you're assumed to be a developer, and then able to fix the rest too. Anyhow I feel that this should rather have been posted in the forum, because you're basically asking for support for some custom way of doing things. You'll find the forum here: https://arastta.org/forum

But regarding SQL changes, they should be very clear to someone pulling code: https://github.com/arastta/arastta/blob/master/install/update.php#L642

**Comment by arnisjuraga** [comment]
Exactly! 
As @syntaxerror mentioned - there could be some database update mechanism dedicated for the developers. If I pull latest changes, should I really manually check any changed database strings?

Couldn't the same `/install/update.php` be modified so that it can execute database changes depending on my current version? (the `VERSION` value should be stored in the Database. Otherwise it's rewritten by `git pull` isn't it?)

**Comment by syntaxerror** [comment]
NO, I did not mention that. Read again please, as this is really going far of topic imho.

Anyhow, as a developer you're also free to propose the needed code changes, submitting your PR's for it.

**Comment by arnisjuraga** [comment]
Ok, anyway - either for Devs from command line, or for users from Admin interface - I just mentioned, that there should be _any_ solution, how to do database upgrades, as there isn't such at the moment.

**Comment by syntaxerror** [comment]
Nor is it in your fork, where you need it ...

Anyway, yes there is automatic database upgrade in every single update, as long as you run the update from admin as you should with Arastta: https://arastta.org/docs/update
